interp/constrexpr_ops.ml

```diff
@@ -64,7 +64,7 @@ let eq_located f (_, x) (_, y) = f x y
 
 let rec cases_pattern_expr_eq p1 p2 =
   if CAst.(p1.v == p2.v) then true
-  else match CAst.(p1.v, p2.v) with
+  else match CAst.get p1, CAst.get p2 with
```

Comment:
I am not opposed to this, but just checking that this is not a leftover from the previous version as the diff could be made smaller by omitting this line.